cypress/cypress/e2e/03_submariner_deploy.cy.js

```diff
@@ -22,29 +22,35 @@ describe('submariner - Deployment validation', {
     })
 
     it('test the Submariner add-ons install', { tags: ['deployment'] }, function () {
-        let clusterSetName = Cypress.env('CLUSTERSET')
+        let clusterSetName = Cypress.env('CLUSTERSET')+'11'
         let nattPort = Cypress.env('SUBMARINER_IPSEC_NATT_PORT')
+        let source = Cypress.env('SUBMARINER_CATALOG_SOURCE')
+        let clusters_list = Cypress.env('CLUSTER_LIST')
+        
         clusterSetMethods.createClusterSet(clusterSetName) 
-        submarinerClusterSetMethods.manageClusterSet(clusterSetName)
+        submarinerClusterSetMethods.manageClusterSet(clusters_list, clusterSetName)
 
         cy.get('button').contains('Save').click()
         cy.contains('Submariner add-ons', {timeout: 3000}).should('exist').click()
-        cy.get('#install-submariner').click()
+        cy.wait(500)
         cy.get('#install-submariner').click()
         cy.get('#available-clusters-input-toggle-select-multi-typeahead-typeahead').click()
         cy.get('.pf-c-select__menu-item').click({multiple: true})
         cy.get('#globalist-enable').click({ force: true })
         cy.get('button').contains('Next').click()
-        cy.get('#natt-port').type('{selectAll}'+nattPort)
-        for (let i=0; i<2; i++) {
+
+        cy.get('.pf-c-wizard__nav-list').eq(1).children().each(() => {
+            cy.get('#natt-port').type('{selectAll}'+nattPort)
+            cy.get('#isCustomSubscription').click()
+            cy.get('#source').type('{selectAll}'+source)
```

Comment:
Have you tested the successful deployment?
I remember there was a channel parameter as well, but not sure if it's required and automatically handled.